ktlint-ruleset-experimental/src/main/kotlin/com/pinterest/ktlint/ruleset/experimental/AnnotationRule.kt

```diff
@@ -106,6 +115,29 @@ class AnnotationRule : Rule("annotation") {
                 }
             }
         }
+
+        if (node.elementType == FILE_ANNOTATION_LIST) {
+            val lineNumber = node.lineNumber()
+            val next = node.nextSibling {
+                !it.isWhiteSpace() && it.textLength > 0 && !(it.isPartOfComment() && it.lineNumber() == lineNumber)
+            }
+            val nextLineNumber = next?.lineNumber()
+            if (lineNumber != null && nextLineNumber != null) {
+                val diff = nextLineNumber - lineNumber
+                if (diff < 2) {
+                    emit(0, fileAnnotationsShouldBeSeparated, true)
```

Comment:
Hm, what if there's a license header above the file annotation, then I guess `0` offset is not correct anymore? According to [codestyle](https://kotlinlang.org/docs/reference/coding-conventions.html#file-annotations), there can be something on top of the annotation.